rotkehlchen/tests/unit/accounting/test_settings.py

```diff
@@ -27,38 +28,33 @@
 from rotkehlchen.tests.utils.history import prices
 from rotkehlchen.tests.utils.messages import no_message_errors
 from rotkehlchen.types import (
+    AssetAmount,
     Location,
-    Price,
     Timestamp,
     TimestampMS,
-    TradeType,
     deserialize_evm_tx_hash,
 )
 
-history5 = history1 + [Trade(
-    timestamp=Timestamp(1512693374),  # cryptocompare hourly BTC/EUR price: 537.805
+history5 = history1 + create_swap_events(
+    timestamp=TimestampMS(1512693374000),  # cryptocompare hourly BTC/EUR price: 537.805
     location=Location.KRAKEN,
-    base_asset=A_BTC,
-    quote_asset=A_EUR,
-    trade_type=TradeType.SELL,
-    amount=FVal('20'),
-    rate=Price(FVal('13503.35')),
-    fee=None,
-    fee_currency=None,
-    link=None,
-)]
+    event_identifier='',
+    spend=AssetAmount(asset=A_BTC, amount=FVal('20')),
+    receive=AssetAmount(asset=A_EUR, amount=FVal('13503.35') * FVal('20')),
+)
 
 
 @pytest.mark.parametrize('mocked_price_queries', [prices])
 @pytest.mark.parametrize('db_settings', [{
     'include_crypto2crypto': False,
     'include_fees_in_cost_basis': False,
 }])
+@pytest.mark.skip('Discuss what to do with crypto2crypto trades')
```

Comment:
What is the problem? I don't understand. A trade and a swap are the same thing. Also such things need notes. It's easy to miss during reviews